homeassistant/components/isy994/translations/cs.json

```diff
@@ -26,7 +26,7 @@
                     "username": "U\u017eivatelsk\u00e9 jm\u00e9no"
```

Comment:
No need to change these as they will get overwritten by lokalize. We should only update `en.json` and `strings.json` in this repo

Reply:
Reverted